theano/tensor/opt.py

```diff
@@ -265,8 +265,8 @@ def apply(self, fgraph):
                 candidate_inputs = [i for i in xrange(len(node.inputs))
                                     if i not in baseline.values() and
                                     not isinstance(node.inputs[i], Constant) and
-                                    # Is next line costly?
-                                    not fgraph.destroyers(node.inputs[i]) and
+                                    # Is next line costly? (used to be fgraph.get_destroyers)
```

Comment:
I would update the comment to:

the next line should not be costly most of the time.